packer-builds/scripts/ssh-keys.sh

```diff
@@ -2,11 +2,11 @@
 
 # key should exist
 # cat to authorized keys
-cat /home/coleman/.ssh/id_rsa.pub >> /home/coleman/.ssh/authorized_keys
+cat /home/$DIGITALOCEAN_USER/.ssh/id_rsa.pub >> /home/$DIGITALOCEAN_USER/.ssh/authorized_keys
 # change permissions on everything
-chmod 0700 /home/coleman/.ssh
-chmod 0644 /home/coleman/.ssh/authorized_keys
-chmod 0644 /home/coleman/.ssh/id_rsa.pub
+chmod 0700 /home/$DIGITALOCEAN_USER/.ssh
+chmod 0644 /home/$DIGITALOCEAN_USER/.ssh/authorized_keys
+chmod 0644 /home/$DIGITALOCEAN_USER/.ssh/id_rsa.pub
 
 # restart sshd
 service sshd restart
```

Comment:
This looks good, but we'll need to pass this var to the script in the packer build via `environment_vars` in the Packer config. See docs for "shell" provisioner: https://www.packer.io/docs/provisioners/shell.html